render.c

```diff
@@ -51,6 +52,8 @@ static cairo_subpixel_order_t get_cairo_subpixel_order(
 		return CAIRO_SUBPIXEL_ORDER_VRGB;
 	case WL_OUTPUT_SUBPIXEL_VERTICAL_BGR:
 		return CAIRO_SUBPIXEL_ORDER_VBGR;
+	default:
+		assert(0);
```

Comment:
Does it fix your problem if you put the `assert(0)` outside of the `switch`?

```c
switch (…) {
case …:
    return …
}
assert(0);
```

You can edit your latest commit by doing:

```
git commit -a --amend
git push --force
```

Reply:
It does solve my compilation error.
Thanks for the git tip.